tests/infra/ccf.py

```diff
@@ -213,6 +213,15 @@ def create_node(self, node_id, host, debug=False, perf=False, recovery=False):
     def remove_node(self):
         last_node = self.nodes.pop()
 
+    def add_node(self, new_node_info, already_exists=False):
```

Comment:
I think it would be better for the python ccf API to stay simple/unaware of whether a node already exists or not. Could `already_exists` be removed as an argument and instead we return an error that the caller needs to handle accordingly?

Reply:
It actually should not exist at all, thanks for noticing